include/TPP/Dialect/Tpp/TppOps.td

```diff
@@ -108,16 +108,15 @@ def Tpp_ReluOp : Tpp_Op<"relu", [SameTypeOperands]> {
 
     ```mlir
  
-    tpp.relu ins(%1: memref<2x2xf32>) out(%2: memref<2x2xf32>)
+    tpp.relu ins(%1: memref<2x2xf32>)
 
     ```
   }];
 
-  let arguments = (ins TppOperand:$input, TppOperand:$output);
+  let arguments = (ins TppOperand:$input);
 
   let assemblyFormat = [{
-      `ins` `(` $input `:` type($input) `)` 
-      `out` `(` $output `:` type($output) `)` attr-dict
+      `ins` `(` $input `:` type($input) `)`  attr-dict
```

Comment:
Yes, this is what I had in mind. The direction looks good to me. I am not sure we want to use `ins` as the tensor is both the input *and* the output. Maybe `inout` but I don't like it much

Reply:
I don't see inout defined in opbase.td, introducing it will need changes to LLVM that needs to be upstreamed, yes?

Reply:
`ins` is just a string that will get printed, we can call it as we want. Similar for `input` we can have `(ins TppOperand:$inout)`.

Reply:
We don't need any upstream extension.